components/pillar-page/block.tsx

```diff
@@ -42,9 +44,12 @@ export default function ProblemBlocks(props: {
               <h3 className="h3 text-xl text-gray-600 flex justify-center items-center">
                 &nbsp; {props.subtitle} &nbsp;
               </h3>
-            </div>
+            </div> 
+          </div>
+          <div className="flex flex-row justify-center gap-40 text-gray-600 mb-16">
```

Comment:
please decrease the margin-bottom and remove this class from h1 'whitespace-nowrap'

Reply:
fixed it!